### PR TITLE
Release v0.4.8

### PR DESCRIPTION
**Pull request by izakp**
cc @dblandin @dleve123

### CODE REVIEW

**Comment by dblandin** [comment]
Nice! Did you see this comment: https://github.com/artsy/hokusai/pull/109#discussion_r235085965?

I think a partial revert of #107 made its way in due to some missing upstream changes in #109.

**Comment by izakp** [comment]
@dblandin ach did not see that - rebased and removed the extra requires in https://github.com/artsy/hokusai/pull/111

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-HOKUSAI
+HOKUSAI [![CircleCI](https://circleci.com/gh/artsy/hokusai/tree/master.svg?style=svg)](https://circleci.com/gh/artsy/hokusai/tree/master)
```
Comment (dblandin) [inline README.md]: Nice :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-## v0.4.7
+## v0.4.8
```
Comment (dleve123) [inline CHANGELOG.md]: Is it worth documenting somewhere in the changelog that 0.4.7 was a bad release? I know that this is intended to be a release PR, but figured it was worth raising this topic in the event that it hasn't been already @izakp

Comment (izakp) [reply CHANGELOG.md]: Ah yeah good idea

Comment (izakp) [reply CHANGELOG.md]: https://github.com/artsy/hokusai/commit/13ad18c52d01b24261014b45f6d53e06841ca239

**Pull request merged**